Assembly-CSharp/Logger.cs

```diff
@@ -180,17 +180,18 @@ public static void LogError(object message)
             Log(message, LogLevel.Error);
         }
 
-        /// <summary>
-        ///     Locks file to write, writes to file, releases lock.
-        /// </summary>
-        /// <param name="text">Text to write</param>
-        private static void WriteToFile(string text)
+		/// <summary>
+		///     Locks file to write, writes to file, releases lock.
+		/// </summary>
+		/// <param name="text">Text to write</param>
+		/// <param name="level">Level of log</param>
+		private static void WriteToFile(string text, LogLevel level)
```

Comment:
why the indentation change

Reply:
wierd. It doesn't show for me like this locally